homework/Contributor.js

```diff
@@ -13,7 +13,37 @@ class Contributor {
    * @param {HTMLElement} container The container element in which to render the contributor.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    // Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const contributorsDiv = Util.createAndAppend('div', container);
+
+    const contributorsLists = Util.createAndAppend('ul', contributorsDiv, { id: 'conts-list' });
+
+    const contributorListItem = Util.createAndAppend('li', contributorsLists, {
```

Comment:
on line 17 of this file you are creating a `div` as a container, then line 19 you are creating a `ul` as a contributors list**s**, then here on line 21 you are creating an `li` element to have at the end the following from the previous comment:
```HTML
<div>
  <ul>
    <li>contributor #1</li>
  </ul>
  <ul>
    <li>contributor #2</li>
  </ul>
  <!-- till the end of our contributors count -->
</div>
```

A `ul` is for declaring lists on the web page.
when you use it we used to tell the browser and the user `Hey I have a list over here` making the browser knows how to style it, and the user can connect it as it's relations for legibility.